go/client/cmd_pgp_encrypt.go

```diff
@@ -15,13 +15,13 @@ import (
 	rpc "github.com/keybase/go-framed-msgpack-rpc"
 )
 
-func NewCmdPGPEncrypt(cl *libcmdline.CommandLine) cli.Command {
+func NewCmdPGPEncrypt(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
```

Comment:
is there a better way to do this in go? like actual dependency injection vs having to pass this everywhere?